types/highland/index.d.ts

```diff
@@ -888,8 +889,28 @@ declare namespace Highland {
 		 * @param {Function} f - the function to handle an error
 		 * @api public
 		 */
-		stopOnError(f: (err: Error) => void): Stream<R>;
-
+    stopOnError(f: (err: Error) => void): Stream<R>;
+    
+    /**
+		 * [splitBy](splitBy) over newlines.
+		 *
+		 * @id split
+		 * @section Streams
+		 * @name Stream.split(s)
+		 * @api public
+		 */
+    split(): Stream<R>;
+      
+    /**
+		 * Splits the source Stream by a separator and emits the pieces in between, much like splitting a string.
+		 *
+		 * @id splitBy
+		 * @section Streams
+		 * @name Stream.splitBy(sep)
+		 * @param {String|RegExp} sep - String | RegExp - the separator to split on
+		 * @api public
+		 */
+    splitBy(sep: string): Stream<R>;
```

Comment:
Tiny nitpick - could you add a blank line between this definition and the next one?

Reply:
will fix